packages/e2e-tests/src/specs/editor/shopping/woocommerce.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import {
+  addRequestInterception,
+  createNewStory,
+  withExperimentalFeatures,
+  publishStory,
+} from '@web-stories-wp/e2e-test-utils';
+
+describe('Shopping', () => {
+  describe('Shopping schema', () => {
+    let stopRequestInterception;
+    withExperimentalFeatures(['shoppingIntegration']);
+
+
+    beforeAll(async () => {
+      await page.setRequestInterception(true);
+      stopRequestInterception = addRequestInterception((request) => {
+        // ... grab story json to validate against schema ?
```

Comment:
@swissspidy @spacedmonkey 

Is there a call / endpoint I can intercept here to get the story config to compare against ?

I can see I'm getting data (via the story dev tools) after I run this e2e test

```
"type": "product",
          "product": {
            "productId": "wc-36",
            "productTitle": "T-Shirt with Logo",
            "productBrand": "",
            "productPrice": "18",
            "productPriceCurrency": "USD",
            "productImages": [],
            "aggregateRating": {
              "ratingValue": "0",
              "reviewCount": 0,
              "reviewUrl": "http://localhost:8899/product/t-shirt-with-logo"
            },
            "productDetails": "This is a simple product.",
            "productUrl": "http://localhost:8899/product/t-shirt-with-logo"
          },
```